tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -206,6 +206,7 @@ Validate data block is in list
     user checks table column heading contains    1    4    Created date    testid:dataBlocks
     user checks table column heading contains    1    5    Actions    testid:dataBlocks
 
+    user reloads page
```

Comment:
I'm not sure about reloading the page here and in the other places, it could disguise problems that occur when you're navigating around the app that don't occur when you have a fresh page load.

Reply:
For some reason after saving data block, robot tests are unable to read the value inside the table. Due which ui tests are failing abnormally. The fix for this kind of problems is to refresh the page